### PR TITLE
RELATED: RAIL-3942, RAIL-3946 Add sdk-ui-dashboard dependency

### PR DESCRIPTION
**Pull request by no23reason**
Make the bootstrapped application ready to use the Dashboard component:
* add dependency and import styles correctly
* use the version of sdk-ui-dashboard that works with CRA
* fix styling issues with the header
* remove useless @types dependencies

JIRA: RAIL-3942, RAIL-3946

### CODE REVIEW

**Comment by yenkins** [comment]
## The code coverage is **the same** after your change 🤟
This is a good thing if you are not sure ;)
#### Report (compared to build [#305](https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/305/))
Covered lines: **22.73%**, change: **0%**
Covered branches: **33.33%**, change: **0%**
#### Coverage report
https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/306/artifact/ci/results/coverage/lcov-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/306/ : SUCCESS in 1m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-oss-check/249/ : SUCCESS in 8s

File: bootstrap/src/components/Header/Header.module.scss [hunk 1]
```diff
@@ -6,11 +6,11 @@
     top: 0px;
     right: 0px;
     left: 0px;
-    min-height: 4em;
+    min-height: $header-height;
     display: flex;
     flex-direction: row;
     align-items: stretch;
-    z-index: 100;
+    z-index: 101; // must be higher than z-index of the sdk-ui-dashboard header (which is 100 currently)
```
Comment (plavemar) [inline bootstrap/src/components/Header/Header.module.scss:13]: Would not be better to set this z-index even higher so we don't need to take care of every possible change of z-index of sdk-ui-dashboard header component?

**Review by plavemar — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
## The code coverage is **the same** after your change 🤟
This is a good thing if you are not sure ;)
#### Report (compared to build [#305](https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/305/))
Covered lines: **22.73%**, change: **0%**
Covered branches: **33.33%**, change: **0%**
#### Coverage report
https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/307/artifact/ci/results/coverage/lcov-report/index.html

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-merge-compliance-check/122/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/307/ : SUCCESS in 48s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-fossa-scanning-zuul-el7/87/ : FAILURE in 5m 23s (non-voting)

**Pull request merged**